src/pages/static/CodeOfConduct.md

```diff
@@ -0,0 +1,24 @@
+## Code of Conduct
+
+At Coding Coach, we value all our mentors, mentees, and contributors. Our goal is to make this a safe space where new and experienced developers can connect and learn from one another. Therefore, we do not tolerate any form of harassment or prejudice. This Code of Conduct applies to all Coding Coach spaces including but not limited to slack, email correspondence, and social media.
+
+Harassment includes but is not limited to:
+  * Offensive comments related to gender, gender identity and expression, sexual orientation, disability, mental illness, physical appearance, body size, age, race, or religion.
+  * Deliberate misgendering or use of ‘dead’ or rejected names.
+  * Physical contact and simulated physical contact (eg, textual descriptions like “*hug*” or “*backrub*”) without consent or after a request to stop.
+  * Threats of violence.
+  * Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm.
+  * Deliberate intimidation.
+  * Stalking or following.
+  * Harassing photography or recording, including logging online activity for harassment purposes.
+  * Unwelcome sexual attention such as sexual images or behaviour in spaces where they’re not appropriate.
+  * Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+  * Continued one-on-one communication after requests to cease.
+  * Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.
+  * Publication of non-harassing private communication.
+
+Members who violate this code of conduct will be approached by someone of the Coding Coach leadership team and asked to stop immediately. Members may also be banned from the Coding Coach slack, blocked on social media, and removed from the website. 
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. To report an incident of harassment you can either directly message one of leadership team, _[names]_, on slack. If you would rather rename anonymous please fill out _[our report form]_. 
```

Comment:
I think we should use google docs for anonymous reports, maybe later we can build something else but for now that should be more then enough

Reply:
Love the idea! I'll set it up!

Reply:
Here is the link to report an incident: https://forms.gle/bcSWqNNcsdo3zDD17